frontend/src/__tests__/components/home.test.js

```diff
@@ -0,0 +1,129 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import Home from '../../../pages';
+import { useGetBooks } from '../../../api/books';
+import userEvent from '@testing-library/user-event';
+jest.mock('../../../api/books');
+jest.mock('../../../api/authors');
+jest.mock(
+  'next/image',
+  () =>
+    function Image({ src, alt, width, height }) {
```

Comment:
Why use of the `function`? Just curious why you keep swapping out `const` and `function`. Right above this line you use an arrow function.

Reply:
was unfamiliar with the proper syntax with mock and arrow functions. I converted it properly now: 

jest.mock('next/image', () => ({ src, alt, width, height }) => {
  // eslint-disable-next-line @next/next/no-img-element
  return (
    // eslint-disable-next-line
    <img src={src} alt={alt} style={{ width: width, height: height }} />
  );
});